cicd/2-cicd/deploy-cicd.sh

```diff
@@ -31,7 +31,7 @@ else
   fi
 fi
 
-ENVIRONMENT_TYPE=${ENVIRONMENT_TYPE-'standard'}
+ENVIRONMENT_TYPE=${ENVIRONMENT_TYPE-'production'}
```

Comment:
should the default be production?

Reply:
Realized this was confusing. Wondering if "adhoc" or "test" would make more sense as the default vs production

Reply:
We could also require it to be specified, and not provide a default. I'd like to tackle that change in a separate PR though.

Reply:
makes sense, I'm fine with this for now